src/utils/getPreparedPeople.tsx

```diff
@@ -0,0 +1,46 @@
+import { getSortPreparedPeople } from './getSortedPeople';
+import { Person } from '../types';
+
+export const getPreparedPeople = (
+  people: Person[],
+  query: string,
+  sex: string,
+  centuries: string[],
+  sort: string,
+  order: string,
+) => {
+  let preparedPeople = [...people];
```

Comment:
The `filter` method returns a copy of the array, so you don't need to do it manually.